src/CommunityToolkit/Xamarin.CommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -60,9 +60,25 @@
     <UseWPF>true</UseWPF>
     <UseWindowsForms>true</UseWindowsForms>
   </PropertyGroup>
+  <PropertyGroup Condition=" $(TargetFramework.StartsWith('netstandard')) ">
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
+  </PropertyGroup>
```

Comment:
@ahoefling this is the bad boy that doesn't allow us to debug that Navigation method haha. Can you remove it please?
```suggestion
```

Reply:
Thanks for finding this, that has been so frustrating. I'll take a look in a little bit

Reply:
I added this when I implemented the PlatformSpecific apis in the NavigationExtensions. After commenting these out, I get the following errors. I'm digging into this to see how I can resolve it

![image](https://user-images.githubusercontent.com/17751436/104113990-47203b00-52cd-11eb-80b1-d9513f8340a2.png)

Reply:
Now it is coming back to me, there is a problem in the csproj around windows libs and netstandard that generates multiple attributes. If you comment out the following statement that proposed change builds
```
<TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(TargetFrameworks);uap10.0.19041;netcoreapp3.1;net472;net471</TargetFrameworks>
```

Reply:
@ahoefling can you check if those errors appear on VS?

Reply:
If the project runs as expected on VS, I believe that is an issue with MSBuild and Rider, sadly we can't use Microsoft.SDK project style until .NET6

Reply:
I saw these errors in Visual Studio last time I was working on this. To be safe I just took a 2nd look in Visual Studio and got the same errors. I tried a few runs of cleaning the solution and deleting the bin/obj dirs. I got these errors when attempt to rebuild the entire solution or just the XCT project
![image](https://user-images.githubusercontent.com/17751436/104114223-e5150500-52cf-11eb-9025-c9ba254e2d19.png)

I think there is an issue with the multi-targeting of the project. I wonder if when you are building if it is somehow only building the android project of XCT and nothing else.

Reply:
@pictos I figured out the problem and it was related to what you mentioned to me earlier in the discord channel. You originally pointed out that the `obj` directory was appearing in the XCT solution. This is where the intermediate files are being generated. Since XCT is multi-targeting so many different target frameworks files are being placed in the obj file. When we compile against `net471` it uses the intermediate files from the `obj` directory as part of the source code. The original solution was prevent them from generating, but that caused issues with the debugger not connecting.

I came across this visual studio question which pointed me towards the direction of just excluding the `obj` directory from compilation
- https://developercommunity.visualstudio.com/content/problem/1055833/duplicate-globalsystemruntimeversioningtargetframe.html

My initial test appeared to work and I could debug the android native renderer again. I ended up moving the statements to exclude the `obj` folder for any netstandard project. This appears to fix the problem. so we can move forward. I believe this is a bigger problem upstream with multi-targeting and how msbuild/dotnet compile the project.

take a look at my change in the latest code push, and if you don't think it is sufficient, please re-open this thread.